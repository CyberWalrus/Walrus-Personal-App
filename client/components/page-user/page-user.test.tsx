import * as Enzyme from "enzyme";
import { shallow } from "enzyme";
import * as Adapter from "enzyme-adapter-react-16";
import toJson, { Json } from "enzyme-to-json";
import * as React from "react";
import PageUser from "./page-user";

Enzyme.configure({ adapter: new Adapter() });
it(`PageUser correctly render`, (): void => {
  const tree: Json = toJson(shallow(<PageUser />));
  expect(tree).toMatchSnapshot();
});
