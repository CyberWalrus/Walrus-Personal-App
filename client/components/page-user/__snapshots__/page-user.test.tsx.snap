// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageUser correctly render 1`] = `
<Switch>
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(ScreenUsers)",
        "type": [Function],
      }
    }
    exact={true}
    path="/user"
  />
  <Route
    path="/user/:id"
    render={[Function]}
  />
</Switch>
`;
